app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee

```diff
@@ -604,7 +604,16 @@ ShortFormApplicationService = (
 
   Service.authorizedToProceed = (toState, fromState, toSection) ->
     return true unless toState && fromState
-    return false unless Service.userCanAccessSection(toSection.name)
+    if not Service.userCanAccessSection(toSection.name)
+      Raven.captureMessage('User attempted to access unauthorized section', {
+        level: 'warning',
+        extra: {
+          toState: toState.url, fromState: fromState,
+          completedSections: Service.application.completedSections,
+          validatedForms: Service.application.validatedForms
```

Comment:
@akegan wondering if it's a good idea to log the whole application itself to see if the data looks legit (e.g. if a section is marked as incomplete and actually is from the values submitted etc).

Reply:
is it OK to log people's personal info in sentry?

here's an example screenshot:
![image](https://user-images.githubusercontent.com/11825994/42651697-ff6d7430-85c4-11e8-8e33-a978ff6ba785.png)

Reply:
updated to include application details. i think we need to revisit in the future and potentially implement scrubbing for sensitive fields